more-python-for-beginners/08 - Managing external resources/demo.py

```diff
@@ -1,8 +1,8 @@
 try:
-	stream = open('output.txt', 'wt')
-	stream.write('Lorem ipsum dolar')
+    stream = open('output.txt', 'wt')
+    stream.write('Lorem ipsum dolar')
 finally:
-	stream.close() # THIS IS REALLY IMPORTANT!!
+    stream.close() # THIS IS REALLY IMPORTANT!!
 
 # with open('output.txt', 'wt') as stream:
-# 	stream.write('Lorem ipsum dolar')
+# stream.write('Lorem ipsum dolar')
```

Comment:
There was some indent in the comment to show line 8 is something within line 7 comment.
Please make it again(maybe 3 -> 4 spaces would be better)

Reply:
Oh! I just used vim editor indenting... Thanks for reviewing!